spring-web/src/main/java/org/springframework/web/context/request/WebRequest.java

```diff
@@ -223,7 +223,7 @@ public interface WebRequest extends RequestAttributes {
 	 * also with conditional POST/PUT/DELETE requests.
 	 * <p><strong>Note:</strong> The HTTP specification recommends
 	 * setting both ETag and Last-Modified values, but you can also
-	 * use {@code #checkNotModified(String)} or
+	 * use {@link #checkNotModified(String)} or
```

Comment:
The use of `@code` is done on purpose, as a `@link` to the current method generates a warning (in IntelliJ IDEA at least), so I will keep this part as it is.

Reply:
Thank you for your feedback!!

But,,, I think this is an overloaded method and not the same one, so there’s no warning in IntelliJ.

Reply:
Good catch, I checked the wrong one, I will bring back this change via an additional commit.

Reply:
Thank you!!!